targetplatform/smarthome.target

```diff
@@ -55,6 +55,7 @@
 <unit id="org.codehaus.jackson.mapper.source" version="1.6.0.v20101005-0925"/>
 <unit id="org.hamcrest.core" version="1.3.0.v201303031735"/>
 <unit id="org.hamcrest.core.source" version="1.3.0.v201303031735"/>
+<unit id="org.json" version="1.0.0.v201011060100"/>
```

Comment:
Why have you changed the target platform? Cannot found a new import that contains "json" in your diff.